### PR TITLE
unit-tests added

### PR DESCRIPTION
**Pull request by SpiderDan98**
- Update unit tests
- Update code style

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,8 +10,13 @@
   ],
   "require": {
     "php": ">=7.1",
-    "spryker/company-business-unit": "^2.0.0",
-    "spryker/company-unit-address": "^1.0.0"
+    "fond-of-spryker/companies-company-users-rest-api": "dev-master",
+    "fond-of-spryker/companies-rest-api": "dev-master",
+    "fond-of-spryker/country": "dev-master",
+    "spryker/company-business-unit": "^1.0.0 || ^2.0.0",
```
Comment (SpiderDan98) [inline composer.json:16]: Checked for possible versions

File: src/FondOfSpryker/Client/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 2]
```diff
@@ -10,6 +10,8 @@
 class CompaniesCompanyAddressesRestApiFactory extends AbstractFactory
 {
     /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Client/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Client/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 3]
```diff
@@ -18,9 +20,9 @@ public function createZedCompaniesCompanyAddressesRestApiStub(): CompaniesCompan
     }
 
     /**
-     * @return \FondOfSpryker\Client\CompaniesCompanyAddressesRestApi\Dependency\Client\CompaniesCompanyAddressesRestApiToZedRequestClientInterface
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Client/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 4]
```diff
@@ -66,12 +73,18 @@ public function getCompanyClient(): CompanyClientInterface
...
+     *
+     * @return \Spryker\Client\CompanyUser\CompanyUserClientInterface
+     */
     public function getCompanyUserClient(): CompanyUserClientInterface
     {
         return $this->getProvidedDependency(CompaniesCompanyAddressesRestApiDependencyProvider::CLIENT_COMPANY_USER);
     }
 
     /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 5]
```diff
@@ -66,12 +73,18 @@ public function getCompanyClient(): CompanyClientInterface
         return $this->getProvidedDependency(CompaniesCompanyAddressesRestApiDependencyProvider::CLIENT_COMPANY);
     }
 
+    /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 6]
```diff
@@ -49,15 +54,17 @@ public function getCompanyBusinessUnitClient(): CompanyBusinessUnitClientInterfa
...
-     * @return \Spryker\Client\Company\CompanyClientInterface
+     * @return \Spryker\Client\CompanyUnitAddress\CompanyUnitAddressClientInterface
      */
     public function getCompanyUnitAddressClient(): CompanyUnitAddressClientInterface
     {
         return $this->getProvidedDependency(CompaniesCompanyAddressesRestApiDependencyProvider::CLIENT_COMPANY_UNIT_ADDRESS);
     }
 
     /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 7]
```diff
@@ -49,15 +54,17 @@ public function getCompanyBusinessUnitClient(): CompanyBusinessUnitClientInterfa
...
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php [hunk 8]
```diff
@@ -40,6 +44,7 @@ public function createCompanyUnitAddressWriter(): CompanyUnitAddressWriterInterf
...
```
Comment (daniel-rose) [inline src/FondOfSpryker/Glue/CompaniesCompanyAddressesRestApi/CompaniesCompanyAddressesRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Zed/CompaniesCompanyAddressesRestApi/Business/CompaniesCompanyAddressesRestApiBusinessFactory.php [hunk 9]
```diff
@@ -18,7 +19,9 @@
 class CompaniesCompanyAddressesRestApiBusinessFactory extends AbstractBusinessFactory
 {
     /**
-     * @return \FondOfSpryker\Zed\CompanyUnitAddressesRestApi\Business\CompanyUnitAddress\CompanyUnitAddressReaderInterface
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompaniesCompanyAddressesRestApi/Business/CompaniesCompanyAddressesRestApiBusinessFactory.php]: remove @throws

File: src/FondOfSpryker/Zed/CompaniesCompanyAddressesRestApi/Business/CompaniesCompanyAddressesRestApiBusinessFactory.php [hunk 10]
```diff
@@ -50,6 +53,8 @@ public function createCompanyUnitAddressWriter(): CompanyUnitAddressWriter
     }
 
     /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompaniesCompanyAddressesRestApi/Business/CompaniesCompanyAddressesRestApiBusinessFactory.php]: remove @throws

**Review by daniel-rose — CHANGES_REQUESTED** [review]

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**